sdk/tests/deqp/functional/gles3/es3fTextureFormatTests.js

```diff
@@ -307,7 +307,12 @@ es3fTextureFormatTests.TextureCubeFormatCase.prototype.testFace = function(face)
         this.m_texture.getRefTexture(), texCoord, renderParams);
 
     // Compare and log.
-    var isOk = glsTextureTestUtil.compareImages(referenceFrame, renderedFrame, threshold);
+    // Skip top right pixel due to Mac Intel driver bug.
+    // https://github.com/KhronosGroup/WebGL/issues/1819
+    var skipPixels = [
+        [this.m_width - 1, this.m_height - 1]
+    ];
```

Comment:
Can skipPixels be applied only when an integer format is used? It would be best to minimize the number of situations where testing is reduced, if most formats are working on this GPU.